tidy3d/components/data.py

```diff
@@ -401,8 +401,10 @@ def apply_syms(self, grid_dict: Dict[str, Coords], sym_center: Coordinate, symme
                 new_data = new_data.interp({dim_name: coords_interp}, kwargs={"fill_value": 0.0})
                 new_data = new_data.assign_coords({dim_name: coords})
 
-                # Apply the correct +/-1 for the data_key component
-                new_data[{dim_name: flip_inds}] *= sym * self._sym_dict[data_key][dim]
+                sym_eval = self._sym_dict.get(data_key)
+                if sym_eval:
```

Comment:
`if sym_eval is not None` is recommended here to be explicit.  For example, what if it's 0?

Reply:
Technically it should be a list, but I agree.